src/sentry/static/sentry/app/components/links/listLink.tsx

```diff
@@ -37,7 +34,14 @@ class ListLink extends React.Component<Props> {
   isActive = () => {
     const {isActive, to, query, index} = this.props;
 
-    return (isActive || this.context.router.isActive)({pathname: to, query}, index);
+    if (typeof to !== 'string') {
```

Comment:
Would it be better to check for undefined? `to` might be a function or `LocationDescriptor`.

Reply:
`isActive` only takes a string. so this likely just wasn't working if it was a function.